src/runtime_strings.js

```diff
@@ -1,38 +1,3 @@
-/** @type {function(number, number=)} */
```

Comment:
It may be worth replacing this with
```
#if ASSERTIONS
function Pointer_stringify() {
  abort("this function has been removed - you should use [..]");
}
#endif
```
which would help debug builds direct people to what they need to do.